src/screens/LogInScreen.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from "react";
+import firebase from "../Firebase";
+import ErrorMessage from "../components/ErrorMessage";
+import BackButton from "../components/BackButton/BackButton";
+import StartScreen from "./StartScreen";
+
+//Test comment 
+
+class LogInScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      errorMessage: null,
+      userLoggedIn: false,
+      goBack: false
+    };
+
+    //binding functions
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) {
+    const itemName = e.target.name;
+    const itemValue = e.target.value;
+
+    this.setState({ [itemName]: itemValue });
+  }
+
+  handleSubmit(e) {
```

Comment:
this function is a bit long, and does a few things.
could be split up to be made more readable.